onset_maproom/maproom.py

```diff
@@ -276,7 +412,17 @@ def onset_plots(
         yaxis_title="Probability of Exceeding",
         xaxis_title=f"Onset Date [days since {search_start_day} {search_start_month}]",
     )
-    return onsetDate_graph, probExceed_onset, None
+    if pd.isnull(onsetDate["onset"].iloc[-1]):
+        germ_sentence = (
+            "Germinating rains have not yet occured as of "
```

Comment:
Is this text correct given the new understanding of the difference between germinating rains and onset date? Seems like this is actually about onset date, not germinating rains.

Reply:
Text correct but data used incorrect